easybuild/easyconfigs/o/Optax/Optax-0.2.2-gfbf-2023a.eb

```diff
@@ -20,13 +20,6 @@ exts_list = [
     ('toolz', '0.12.1', {
         'checksums': ['ecca342664893f177a13dac0e6b41cbd8ac25a358e5f215316d43e2100224f4d'],
     }),
-    ('flit_core', '3.9.0', {
-        'checksums': ['72ad266176c4a3fcfab5f2930d76896059851240570ce9a98733b658cb786eba'],
-    }),
-    ('absl-py', '2.1.0', {
```

Comment:
@lexming Why isn't absl-py` required as dependency if this is removed? Was that incorrect to begin with?